subprojects/performance/src/performanceTest/groovy/org/gradle/performance/regression/android/FasterIncrementalAndroidBuildsPerformanceTest.groovy

```diff
@@ -70,21 +72,22 @@ class FasterIncrementalAndroidBuildsPerformanceTest extends AbstractCrossBuildPe
 
     def "file system watching baseline non-abi change (build comparison)"() {
         given:
+        runner.measureBuildOperation(ExecuteTaskActionBuildOperationType.name)
         runner.buildSpec {
-            displayName("with file system watching")
+            displayName("with file system watching - Gradle 6.7")
             configureForNonParallel(delegate)
             testProject.configureForNonAbiChange(delegate)
             invocation {
                 args.addAll(Optimization.WATCH_FS.arguments)
+                distribution(buildContext.distribution("6.7-rc-2"))
             }
         }
```

Comment:
It would be great to add a no-FSW baseline with Gradle 6.7, too.

Reply:
Added.